taglib/src/test/java/com.reevoo.taglib/ReevooTaglibTest.java

```diff
@@ -24,7 +24,7 @@ public void setUp() throws Exception {
         super.setUp();
         reevooTag.setDynamicAttribute("", "sku", "ABC123");
         reevooTag.setTrkref("FOO");
-        reevooTag.setBaseURI("http://mark.reevoo.com/endpoint");
+        reevooTag.setBaseURI("https://mark.reevoo.com/endpoint");
```

Comment:
Is it not better adding this url without the protocol like ```//mark.reevoo.com/endpoint```. In this way if they use this over http they will get the content over http.